ambry-rest/src/main/java/com.github.ambry.rest/NettyMessageProcessor.java

```diff
@@ -218,7 +223,13 @@ public void channelRead0(ChannelHandlerContext ctx, HttpObject obj) throws RestS
     if (isOpen()) {
       logger.trace("Reading on channel {}", ctx.channel());
       long currentTime = System.currentTimeMillis();
-
+      if (firstMessageReceived.compareAndSet(false, true)) {
+        if (ctx.pipeline().get(SslHandler.class) != null) {
+          nettyMetrics.sslChannelActiveToMessageReceiveTimeInMs.update(currentTime - channelActiveTimeMs);
```

Comment:
shouldn't we name this "channelActiveToFirstMessageReceiveTimeInMs"

Reply:
renamed